README.md

```diff
@@ -217,6 +217,24 @@ Stop the tunnel with: `make <env> stop-tunnel`
 ## Other useful commands
 Type `make` to get the list of all available commands.
 
+## Known caveats
+
+### Fly version
+
+Current mechanism of updating fly binary only checks if the date of your local
+binary is older than one on the server. If you have been using previous fly
+version recently, it won't get updated. In that case, delete the binary in /bin
```

Comment:
When the timestamp of the local binary is more recent than the one hosted on the server, you'll receive a `304 Not Modified` preventing to download a new file. The old (current) version of fly, doesn't support the `sync` functionality, preventing it from doing a self upgrade.

We could download the `fly` binary on each and every run, however this means there is this time and bandwidth "wasted". This should possibly happen, only once - when upgrading from version < `1.0`. After this initial upgrade, the `fly` should either be able to sync itself or download if not existent.